module/commands/help.py

```diff
@@ -80,6 +72,7 @@ def help_dip_cdl(update: Update, context: CallbackContext, chat_id: int, message
     keyboard.append([InlineKeyboardButton(" ~ Dipartimento e CdL ~ ", callback_data="NONE")])
     keyboard.append([
         InlineKeyboardButton("📖 Esami (link)",         callback_data="md_esami_link"),
+        InlineKeyboardButton("📖 Cerca Esami",         callback_data="fr_esami"),
```

Comment:
Of course this still needs to be implemented

Reply:
This is not supposed to be there, it is a test regarding #174